expense-splitter/src/App.jsx

```diff
@@ -24,6 +25,7 @@ function App() {
               <Routes>
```

Comment:
Consider nested routes to keep things a little more readable

For more info: 
https://reactrouter.com/en/main/start/overview#nested-routes